test/test-thread-affinity.c

```diff
@@ -99,6 +101,22 @@ TEST_IMPL(thread_affinity) {
   ASSERT(cpumask[t2first + 2] == (ncpus >= 3));
   ASSERT(cpumask[t2first + 3] == 0);
 
+  c = uv_thread_getcpu();
+  ASSERT(c >= 0);
```

Comment:
Can you use the new-style macros? E.g.:
```suggestion
  ASSERT_GE(c, 0);
```
ASSERT -> ASSERT_EQ